pom.xml

```diff
@@ -81,6 +81,7 @@
   </distributionManagement>
 
 <properties>
+    <revision>7.10.0-SNAPSHOT</revision>
```

Comment:
This is the only place the actual version is specified

Reply:
Yes, that is correct. Either passing it via -D or specifying it here can be used, but I added it here for simplicity.